### ISSUE TITLE
steamshovel - cmake warning for boost 1.38 (Trac #893)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#887]
Have cmake warn us when trying to use boost 1.38 and newer python.

Migrated from https://code.icecube.wisc.edu/ticket/893
```json
{
    "status": "closed", 
    "changetime": "2015-04-08T17:24:35", 
    "description": "Have cmake warn us when trying to use boost 1.38 and newer python.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1428513875779270", 
    "component": "combo core", 
    "summary": "steamshovel - cmake warning for boost 1.38", 
    "priority": "major", 
    "keywords": "steamshovel", 
    "time": "2015-03-03T23:26:27", 
    "milestone": "", 
    "owner": "nega", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150303T23:28:39: 
* david.schultz changed owner from "david.schultz" to "nega"
* david.schultz changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20150408T16:55:25: nega commented:

> Test references >= 2.7.3. #warning references >= 2.7.4. Which is correct?

**Comment by icecube-trac** [comment]
Trac update at 20150408T17:22:36: david.schultz commented:

> 2.7.3 for Ubuntu, 2.7.4 for regular python.  I'd just go with 2.7.3 to be safe.

**Comment by icecube-trac** [comment]
Trac update at 20150408T17:24:35: 
* nega commented:

> In changeset 131214:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="131214"
> add a warning for bad boost+python combo
> 
> closes #887
> ```


* nega changed resolution from "" to "fixed"
* nega changed status from "assigned" to "closed"
